libs/common/src/schemas/tfm/create-user-request.schema.ts

```diff
@@ -0,0 +1,19 @@
+import z from 'zod';
+import { TfmTeamSchema } from './tfm-team.schema';
+
+/**
+ * Used during the SSO login process when a user is required to be created in TFM
+ * It is used as a foundation to the upsert user request
+ * @see UPSERT_USER_REQUEST_SCHEMA for the upsert user request schema this create user request schema influences
+ * @see UPDATE_USER_REQUEST_SCHEMA for the update user schema this create user request schema influences
+ */
+export const CREATE_USER_REQUEST_SCHEMA = z.object({
+  azureOid: z.string(),
+  email: z.string(),
+  username: z.string(),
+  teams: z.array(TfmTeamSchema),
+  timezone: z.string(),
+  firstName: z.string(),
+  lastName: z.string(),
+  lastLogin: z.number().int().positive(), // unix timestamp
```

Comment:
There's already 2x different definitions of `lastLogin` with zod - this introduces a new 3rd definition, can we update to be the same?

```ts
lastLogin: z.number().optional(), // central API
lastLogin: z.number(), // libs/common
lastLogin: z.number().int().positive(), // libs/common
```

Reply:
updating! thanks :)

Reply:
Commonised this the best I can. Also added in a `UnixTimestampMilliseconds` as we should be descriptive with our units -- as currently it leads to confusion and we cannot be sure what format a timestamp is in as a result.